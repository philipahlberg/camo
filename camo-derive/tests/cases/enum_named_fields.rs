use camo_derive::Camo;

#[derive(Camo)]
enum Foo {
    One { a: i32, b: i32 },
}

fn main() {}
