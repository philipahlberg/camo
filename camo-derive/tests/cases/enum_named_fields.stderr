error: `camo` does not support named fields in enums
 --> tests/cases/enum_named_fields.rs:5:9
  |
5 |     One { a: i32, b: i32 },
  |         ^^^^^^^^^^^^^^^^^^
